graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -20,6 +20,41 @@
 )
 
 
+def _get_inherited_field_types(class_to_field_type_overrides, schema_graph):
+    """Return a dictionary describing the field type overrides in subclasses."""
+    inherited_field_type_overrides = dict()
+    for superclass_name, field_type_overrides in class_to_field_type_overrides.items():
+        for subclass_name in schema_graph.get_subclass_set(superclass_name):
+            inherited_field_type_overrides.setdefault(subclass_name, dict())
+            inherited_field_type_overrides[subclass_name].update(field_type_overrides)
+    return inherited_field_type_overrides
+
+
+def _validate_overriden_classes_have_no_superclasses(class_to_field_type_overrides, schema_graph):
+    """Assert that the fields we want to override are not defined in a superclass."""
+    for class_name, field_type_overrides in six.iteritems(class_to_field_type_overrides):
+        for superclass_name in schema_graph.get_inheritance_set(class_name):
+            if superclass_name != class_name:
+                superclass = schema_graph.get_element_by_class_name(superclass_name)
+                for field_name in field_type_overrides:
+                    if field_name in superclass.properties:
+                        raise AssertionError(
+                            u'Attempting to override field "{}" from class "{}", but the field is '
+                            u'defined in superclass "{}"'
+                            .format(field_name, class_name, superclass_name))
+
+
+def _get_classes_with_no_properties(schema_graph):
+    """Return the set of classes that have no properties. There are not representable in GraphQL."""
```

Comment:
`There are` -> `They are`. Or maybe just remove the sentence. Decide whether this function gets classes with no properties, or classes that can't be represented (I vote for the former). Then make the title, documentation, and name of return value agree on this. Then when you use the function you can comment that currently the only classes we can't represent are those with no properties.